### PR TITLE
mainpage outlook and file uploader basics

### PR DESCRIPTION
**Pull request by Wentao-7114**
First day PR, developed main page, file upload component and basic styling.

### CODE REVIEW

**Comment by CambioAI** [comment]
Can you post future PR in below format

Features:
- Include some screen shots to ease the review process.

Tests:
...

**Comment by Wentao-7114** [comment]
Commited Oct 15
New Feactures: 
1. created the parse document webpage, including extracting full content and extracting key-value pair (unfinished)
image
image
image
image
2. Created reusable components for parse document page, like the left/right/side panels and the file uploader

Next steps:
1. finish the remaining extracting fuctions
2. implement the tour function to guide endusers
3. implement pdf preview page by page if possible
4. further polish the stylings

File: new_frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```
Comment (CambioML) [inline new_frontend/README.md:9]: add a  `npm install` section

File: new_frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+
+If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+
+## Learn More
```
Comment (CambioML) [inline new_frontend/README.md:42]: Let's add a section to explain your project structual looks like.

**Comment by goldmermaid** [comment]
LGTM.

**Pull request merged**